### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by calicodes23** [community/community#29980] [Other Feature Feedback, Questions, & Ideas]
## Achievements

What a fun and pleasant new feature! Love it and wanted to shout out the GH team!

**Comment by KorigamiK** [comment]
It is a cool feature, but I think the icon art is too childish for the platform.

**Comment by NikhilMath** [comment]
I like it too, I am still trying to get the badge for when you answer a question. For some reason I don't seem to be getting lucky for those! lol
